heron/scheduler-core/src/java/BUILD

```diff
@@ -18,10 +18,15 @@ spi_deps_files = [
     "//heron/spi/src/java:utils-spi-java",
 ]
 
+schedulers_deps_files = [
+    "//heron/schedulers/src/java:utils-schedulers-java",
```

Comment:
Good Point!  Thank you for your advice. I'll try to move them into schedulers-core package